pkg/distro/fedora/imagetype.go

```diff
@@ -418,11 +418,24 @@ func (t *imageType) checkOptions(bp *blueprint.Blueprint, options distro.ImageOp
 		return []string{w}, nil
 	}
 
-	if customizations.GetInstaller() != nil {
+	if instCust, err := customizations.GetInstaller(); instCust != nil {
 		// only supported by the Anaconda installer
 		if slices.Index([]string{"iot-installer"}, t.name) == -1 {
-			return nil, fmt.Errorf("installer customizations are not supported for %q", t.name)
+			return nil, fmt.Errorf("installer customizations are not supported for %q", t.Name())
 		}
+
+		// NOTE: the image type check is redundant with the check above, but
```

Comment:
Pardon my ignorance, what redundant with what check above exactly? It's not the check from GetInstaller(), iirc/aiui it does not check for user cusomizations.

Reply:
The redundancy is between:
```go
if slices.Index([]string{"iot-installer"}, t.name) == -1 { ... }
```
and the first part in the `&&` chain below
```
if t.Name() == "iot-installer" && ...
```

The first one should list all Anaconda installers but there's an open issue about the `image-installer` (which was buried in a closed PR and I transferred to an issue now #672).
The second check should only be for the `iot-installer`.